### DISCUSSION TITLE
Dropping yED

### DISCUSSION THREAD

**Discussion by iamjuing** [StateSmith/StateSmith#139] [Ideas]
Have you considered dropping support for yED diagrams?
I mean, I tried it and I can not imagine anybody picking that tool for drawing state machines.
It was quite painful to draw even a simple diagram.

My point is ... would it help you to save time?

**Comment by adamfk** [comment]
yeah, yEd is a super pain until you get used to it. I had to make an entire page explaining how to use yEd without going crazy :) Then it isn't too bad. It actually does a few things better than draw.io.

I plan to keep supporting yEd because some of my work projects use it. It was also what we initially launched with. Supporting multiple front ends isn't very difficult because they just convert to some diagram nodes. Then those diagram nodes are converted to state machine vertices. https://github.com/StateSmith/StateSmith/wiki/How-StateSmith-Works

That said, if we add significant new diagram features, they might not be added to yEd.
